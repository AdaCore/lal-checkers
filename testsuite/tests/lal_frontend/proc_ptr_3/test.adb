procedure Main is
   X : Integer;
   P : access procedure;

   procedure G is
   begin
      X := 14;
   end G;

   procedure Test is
   begin
      P := G'Access;
   end Test;
begin
   Test;
end Main;
