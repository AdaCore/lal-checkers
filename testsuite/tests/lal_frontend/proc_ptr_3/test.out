Program:
  $stack#1 = Updated($stack#1, P_ptr#2, Subp_<SubpBody ["G"] test.adb:5:4-8:10>(X_ptr#3))
  end:
